lib/statsd.rb

```diff
@@ -140,7 +201,7 @@ def send(stat, delta, type, sample_rate=1)
       stat.gsub!(/::/, ".".freeze)
       stat.gsub!(RESERVED_CHARS_REGEX, "_".freeze)
 
-      msg = ""
+      msg = String.new
```

Comment:
Correct guess! In 2.3 (if you enable frozen by default), and in 3.0 (for all strings), the empty string will come as frozen, so we won't be able to append to it. This small change will hopefully save us a small version bump! :)

(in the future we should also be able to remove all the `.freeze` calls in the file, but they leaving them there causes no harm at all)